weave/wandb_interface/wandb_stream_table.py

```diff
@@ -195,34 +143,30 @@ def log(self, row_or_rows: ROW_TYPE) -> None:
         for row in row_or_rows:
             self._log_row(row)
 
-    def rows(self) -> graph.Node:
-        if self._weave_stream_table_ref is None:
-            raise errors.WeaveInternalError("ref is None after ensure")
-        return stream_table_ops.rows(
-            weave_api.get(str(self._weave_stream_table_ref.uri))
+    def rows(self) -> None:
+        raise errors.Error(
+            "reading stream tables is not supported in wandb, use weave.StreamTable"
         )
 
-    def _ipython_display_(self) -> graph.Node:
-        from .. import show
-
-        return show(self.rows())
-
     def _log_row(self, row: dict) -> None:
         row_copy = {**row}
         row_copy["_client_id"] = self._client_id
-        if "timestamp" not in row_copy:
-            row_copy["timestamp"] = datetime.datetime.now()
-        payload = row_to_weave(row_copy, self._artifact)
-        self._lite_run.log(payload)
+        self._lite_run.log(row_copy)
 
     def finish(self) -> None:
-        if self._artifact:
-            self._artifact.cleanup()
-        if self._lite_run:
-            self._lite_run.finish()
+        with self._lock:
+            if self._artifact:
+                self._artifact.cleanup()
+            if self._lite_run:
+                self._lite_run.finish()
 
     def __del__(self) -> None:
-        self.finish()
+        try:
+            self.finish()
+        except Exception:
+            # I was seeing exceptions in yea tests, this prevents
+            # ignored exception warnings
+            pass
```

Comment:
The yea tests were complaining about calling a method on None.  I think theres some weird interactions between _at_exit, __del__, and finish.  Anyway, this makes it less scary to users and they can't do anything at this point anyway.